src/Components/TasksCovered/TasksCovered.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import iconList from './iconList';
+import smile1 from '@/images/Homepage/tasksCovered/smile1.svg';
+import smile2 from '@/images/Homepage/tasksCovered/smileDown2.svg';
+import smile3 from '@/images/Homepage/tasksCovered/smile 3.svg';
+import './style.scss';
+
+const TasksCovered: React.FC = () => {
+  return (
+    <div className="container">
+      <section className="section-task">
+        <h2 className="title-task">
+          We've got all
+          <br /> your
+         
+            <span className="title-task-word"> covered</span>
+            <span className="smile-group">
+            <img src={smile1} className="smile" alt="smile" />
+            <img src={smile2} className="smile" alt="smile" />
+            <img src={smile3} className="smile" alt="smile" />
```

Comment:
try to create a better code - so you dont repeat yourself 3 times with basically same img tag definition.

Reply:
Added arr with images and changed to { imageArr.map(item => (
  <img 
    key={item.id} 
    src={item.imageUrl} 
    alt={item.imageTitle} 
    className="smile" 
  />
))}